user/dependency_management/gradle-plugin.md

```diff
@@ -43,6 +43,14 @@ gauge {
     additionalFlags = '--verbose'
 }
 
+// copying required dependencies for Gauge (required)
+task copyLibs {
+    copy {
+        from configurations.runtime
+        into "$projectDir/libs"
+    }
+}
+
```

Comment:
I'm on it. Might take some time. It's an intermediate release which contains some major bug fixes. I'll publish new version soon